mesh/profiles/urls.py

```diff
@@ -18,6 +19,8 @@
 
     # GET /profiles/user-name/:account_id
     # POST /profiles/user-name/:account_id
+    path("biography/<int:account_id>", profile_views.BiographyView.as_view()),
+    path("profile-picture/<int:account_id>", profile_views.ProfilePicturesView.as_view(), name="profile-picture"),
```

Comment:
What are these 2 lines doing here?

Reply:
Those are duplicate views. They don't need to be there. Thanks for catching that.